subprojects/docs/src/docs/userguide/dependencyManagementTerminology.adoc

```diff
@@ -0,0 +1,49 @@
+[[dependency_management_terminology]]
+== Dependency Management Terminology
+
+Dependency management comes with a wealth of terminology. Here you can find the most commonly-used terms including references to the user guide to learn about their practical application.
+
+[[sub:terminology_configuration]]
+=== Configuration
+
+A configuration is a named set of <<sub:terminology_dependency,dependencies>> grouped together for a specific goal: For example the implementation configuration represents the set of dependencies required to compile a project. Configurations provide access to the underlying, resolved <<sub:terminology_module,modules>> and their artifacts. For more information, see <<sub:scope_of_dependency_configurations>>.
+
+[NOTE]
+====
+The word "configuration" is an overloaded term and has a different meaning outside of the context of dependency management.
+====
+
+[[sub:terminology_dependency]]
+=== Dependency
+
+A dependency is another piece of software required to build, test or run a <<sub:terminology_module,module>>. For more information, see <<declaring_dependencies>>.
```

Comment:
After our discussions (a dependency itself is not the module/software), I would replace:

_A dependency is another..._

with

_A dependency points at another..._

Reply:
That's not how I read the sentence here. It's just another piece of software. It doesn't really say that it is a module.

Reply:
Sorry that was unclear. I mean a dependency is not _a piece of software_. A dependency is just the pointer at _**some** piece of software_ (which _piece_ that exactly will be yet needs to be resolved).

Reply:
done. https://github.com/gradle/gradle/commit/8736ff06ec65040a90b2a569b0e49196cfbad9a5